homeassistant/components/history_stats/sensor.py

```diff
@@ -62,7 +62,9 @@ def exactly_two_period_keys(conf):
     PLATFORM_SCHEMA.extend(
         {
             vol.Required(CONF_ENTITY_ID): cv.entity_id,
-            vol.Required(CONF_STATE): cv.string,
+            vol.Required(CONF_STATE): vol.Any(
+                cv.string, vol.All(cv.ensure_list, [cv.string])
+            ),
```

Comment:
Why not ensure it is always a list?

```suggestion
            vol.Required(CONF_STATE): vol.All(cv.ensure_list, [cv.string]),
```

Reply:
I wanted to make the change backwards compatible. Is just checking "Breaking change (fix/feature causing existing functionality to break)" in the PR sufficient to justify a breaking change?